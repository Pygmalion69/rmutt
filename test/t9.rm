// regexes
a: "i like to eat apples and bananas" > /[aeiou]+/oo/;