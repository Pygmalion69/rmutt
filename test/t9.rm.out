oo lookoo too oot oopploos oond boonoonoos
